data/en/getlocaleinfo.json

```diff
@@ -4,29 +4,34 @@
 	"syntax":"getLocaleInfo()",
 	"returns":"struct",
 	"related":["getLocaleDisplayName","getLocale","getLocaleCountry","getLocaleLanguage"],
-	"description":"A short description that describes what the tag or function does.",
-	"discouraged":"Only add this key if this tag/function is discouraged by the community.",
+	"description":"Returns a structure containing info to a specific locale.\r\n\r\nThis function combines the locale functions getLocaleCountry, getLocaleDisplayName, and getLocaleLanguage to a single function.",
 	"params": [
-		{"name":"locale","description":"Geographic/language locale value","required":true,"default":"getLocale()","type":"string","values":["en","de_DE","de_ch","..."]},
-		{"name":"displayLocale","description":"Locale's display name/Output language","required":false,"default":"getLocaleDisplayName()","type":"string","values":["English","German","French","..."]}
+		{"name":"locale", "description": "Geographic/language locale value, where the format is a combination of an <a href=\"https://en.wikipedia.org/wiki/ISO_639-1\">ISO 639-1</a> code and an optional <a href=\"https://en.wikipedia.org/wiki/ISO_3166-1\">ISO 3166-1</a> code separated by a dash or an underscore.", "required": false, "default": "getLocale()", "type":"string", "values": ["en","de-DE","de-CH","..."]},
+		{"name":"dspLocale", "description": "Locale's display name/output language", "required": false, "default": "getLocaleDisplayName()", "type": "string", "values": ["English","German","French","..."]}
 	],
 	"engines": {
-		"lucee": {"minimum_version":"5", "notes":"Combines the locale functions <code>getLocaleCountry</code>, <code>getLocaleDisplayName</code>, <code>getLocaleInfo</code>, <code>getLocaleLanguage</code> to a single function", "docs":""}
```

Comment:
Why is that wrong?

Reply:
Three reasons:
1. I've moved that to the general description because I assume the functionality will be the same (combining the data of the other locale functions) when other engines implement it. This also has the nice side effect that the function names are linked to their descriptions automatically.
2. HTML isn't interpreted in engine notes, i.e. `<code>` is output as text. Might be worth another issue though.
3. The note contained `getLocaleInfo` itself, which doesn't make sense.

Reply:
It might be interpreted when it is markdown, but I'm not sure because not everything in the docs is parsed.

But you completely deleted the description oO

Reply:
FYI You can use backticks instead of the `<code>` tag, but I don't think they are necessary for function names as it should link them automatically.